apps/portal/src/app/contracts/sidebar.tsx

```diff
@@ -56,54 +56,56 @@ export const sidebar: SideBar = {
           href: "/contracts/explore/overview",
           name: "Overview",
         },
-        {
-          links: [
-            {
-              links: [
-                {
-                  href: `${prebuiltModular}/token`,
-                  name: "Token",
-                },
-                {
-                  href: `${prebuiltModular}/token-drop`,
-                  name: "Token Drop",
-                },
-              ],
-              name: "ERC-20",
-            },
-            {
-              links: [
-                {
-                  href: `${prebuiltModular}/nft-collection`,
-                  name: "NFT Collection",
-                },
-                {
-                  href: `${prebuiltModular}/nft-drop`,
-                  name: "NFT Drop",
-                },
-                {
-                  href: `${prebuiltModular}/open-edition`,
-                  name: "Open Edition",
-                },
-              ],
-              name: "ERC-721",
-            },
-            {
-              links: [
-                {
-                  href: `${prebuiltModular}/edition`,
-                  name: "Edition",
-                },
-                {
-                  href: `${prebuiltModular}/edition-drop`,
-                  name: "Edition Drop",
-                },
-              ],
-              name: "ERC-1155",
-            },
-          ],
-          name: "Pre-built Modular",
-        },
+        // DEPRECATED: Pre-built Modular contracts have been deprecated from explore
```

Comment:
delete instead of commenting it out, we have git history if we need it back later